.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1 +1,32 @@
-<!-- Please provide a clear and meaningful description to the CHANGELOG.md file if this PR contributes some significant changes -->
+<!--
+Thank you for your Pull Request. Please provide a description above and review
+the requirements below.
+
+Bug fixes and new features should include tests and possibly benchmarks.
+
+Contributors guide: https://github.com/theia-ide/theia/blob/master/CONTRIBUTING.md
+-->
+
+#### What it does
+<!-- Include relevant issues and describe how they are addressed -->
+
+#### How to test
+<!-- Describe how a reviewer can verify changes within Theia repo -->
+
+#### Review Checklist
+<!-- For completed items, change [ ] to [x].  -->
+<!-- For inapplicable items, cross out with `[ ] foo` to `[x] ~~foo~~`.  -->
+
```

Comment:
I would put this first. It might seem obvious but I feel that having to check this explicitly might motivate to go through it once more or extra carefully.